app/models/user.rb

```diff
@@ -82,34 +82,25 @@ def records_not_reviewed
     end
   end
 
-  # ====Params
-  # None
-  # ====Returns
-  # Whether or not the User is an admin.
-  # ==== Method
+  # TODO: are these comments still valid?
   # This method checks if the user is a "admin", which is a legacy role name. It's only
   # intended for backwards compatibility. For new code, use the 'role' attribute directly.
   def admin
-    role.eql?("admin")
+    role == 'admin'
```

Comment:
it seems to me that admin, curator, and daac_curator can all be removed in favor of. using the role property in the model.   maybe before they had some other way for determining if the user was an admin, then finally created a role property?

Reply:
If I were starting from scratch I would want to have the role property, but I would also probably implement these helper methods, especially the `arc?` method. The one thing I would change might be for these two methods to change to `admin?` and `arc_curator?`
So do you think we should remove them or maybe make the change and keep them?

Reply:
think just adding the ? makes sense.   seems like a nice convenience method to just user.admin? and there maybe other logic in the future that might want to check for these kinds of things.

Reply:
I will make the change for the helper methods.

When searching to make the changes, it looks like the Users table used to have an admin field and a curator field, before they added the role field.